### ISSUE TITLE
TypeError: Tensors in list passed to 'values' of 'Pack' Op have types [int32, int32, int32, float32] that don't all match.

### ISSUE THREAD

**Issue by crankson123** [loliverhennigh/Steady-State-Flow-With-Neural-Nets#3]
When I conduct the code, there exist some errors about 'TypeError: Tensors in list passed to 'values' of 'Pack' Op have types [int32, int32, int32, float32] that don't all match.'. I use Python 3.5, so how can I solve this issue, hoping that you can help me.

**Comment by je-santos** [comment]
I'm on the same boat. It might be that we are using python 3 :/

**Comment by loliverhennigh** [comment]
That is correct, I was doing a division and I was expecting an int but it gave a float because of python 3. I just updated this and it should work now. I just tested it with 3.6.

**Comment by je-santos** [comment]
Thanks a lot Oliver! I'll give it a shot.

**Comment by je-santos** [comment]
Hey Oliver, while testing the code I got the following error:

  _File "C:\Users\ME\AppData\Roaming\Python\Python36\site-packages\tensorflow\python\framework\errors_impl.py", line 526, in __exit__
    c_api.TF_GetCode(self.status.status))

NotFoundError: Failed to create a directory: ../checkpoints/min_queue_examples_1000/model_res/nr_res_blocks_1/gated_res_True/nonlinearity_concat_elu/batch_size_8/max_steps_500000/keep_prob_0.7; No such file or directory_

Do you know what might be causing this?

**Comment by xycim** [comment]
@je-santos Hi,do you know how to save this problem
"NotFoundError: Failed to create a directory: ../checkpoints/min_queue_examples_1000/model_res/nr_res_blocks_1/gated_res_True/nonlinearity_concat_elu/batch_size_8/max_steps_500000/keep_prob_0.7; No such file or directory_"?
